resources/js/components/Listing/Listing.vue

```diff
@@ -102,15 +102,20 @@ export default {
                     field: filter.code + (this.filterType(filter) == 'string' ? '.keyword' : ''),
                     type: this.filterType(filter),
                 })),
-                { attribute: 'category_lvl0', field: 'category_lvl0.keyword', type: 'string' },
-                { attribute: 'category_lvl1', field: 'category_lvl1.keyword', type: 'string' },
-                { attribute: 'category_lvl2', field: 'category_lvl2.keyword', type: 'string' },
-                { attribute: 'category_lvl3', field: 'category_lvl3.keyword', type: 'string' },
+                ...this.categoryAttributes.map((attribute) => ({
+                    attribute: attribute,
+                    field: attribute + '.keyword',
+                    type: 'string',
+                })),
             ]
             // TODO: Double check this and how it's used.
             // .concat(this.additionalFilters)
         },
 
+        categoryAttributes() {
```

Comment:
I think we should keep the syntax inline, use this one everywhere; or the syntax used there. Pros/cons?

Reply:
Ah sorry, the name is fine. It's the syntax; 
```
categoryAttributes() {
```
vs
```
categoryAttributes: function () {
```
That's currently mixed.

Reply:
Ah, well we do use the former in most places that I know of.

Looking at the documentation, the Vue3 documentation has the `categoryAttributes() {` format by default whereas (apparently) the Vue2 documentation has the `categoryAttributes: function() {` format. Personally I prefer the former just because it looks cleaner to me. Aside from that I don't think there are any other reasons to use one over the other.

Reply:
Let's use the ES6 shorthand syntax everywhere 🚀 eg: `categoryAttributes() {`